src/components/ec-inline-input-field/ec-inline-input-field.spec.ts

```diff
@@ -69,26 +97,34 @@ describe('EcInlineInputField', () => {
           expect(wrapper.emitted('edit')).toBeUndefined();
           await wrapper.findByDataTest('ec-inline-input-field-value-text__action').trigger('click');
 
-          expect(wrapper.emitted('edit').length).toBe(1);
+          expect(wrapper.emitted('edit')?.length).toBe(1);
```

Comment:
we should use InlineInputEvent enum here. There are couple of places in vue file as well